src/coreclr/jit/lower.cpp

```diff
@@ -3226,7 +3227,53 @@ GenTree* Lowering::LowerJTrue(GenTreeOp* jtrue)
             return nullptr;
         }
     }
-#endif // TARGET_ARM64
+#elif defined(TARGET_LOONGARCH64)
+    GenTree* relop    = jtrue->gtGetOp1();
+    GenTree* relopOp1 = relop->AsOp()->gtGetOp1();
+    GenTree* relopOp2 = relop->AsOp()->gtGetOp2();
+
+    if (relopOp1->IsCnsIntOrI() && relopOp2->IsCnsIntOrI())
```

Comment:
> Seems like this should be deleted.

Thanks. I will delete it.

> Also the code that is trying to contain all constants possible is equally suspicious, reading [the manual](https://loongson.github.io/LoongArch-Documentation/LoongArch-Vol1-EN.html#branch-instructions) it seems LA's branch instructions are very simple and at most support containment of a zero (`SLT[U]I` additionally supports a 12 bit immediate, still, that's not "any 64 bit constant" that this code is containing).

If meeting this, it will be moved to a register to avoid not fit for one instruction.